tools/test/examples/examples_lib.py

```diff
@@ -382,27 +382,36 @@ def compile_repos(config, toolchains, targets, profiles, verbose, exp_filter, jo
                     summary_string = "%s %s %s" % (name, target, toolchain)
                     logging.info("Compiling %s" % summary_string)
 
-                    build_command = ["mbed-cli", "compile", "-t", toolchain, "-m", target, "-j", str(jobs)] + (['-vv'] if verbose else [])
-                    if profiles:
-                        for profile in profiles:
-                            build_command.extend(["--profile", profile])
-
-                    logging.info("Executing command '%s'..." % " ".join(build_command))
-                    proc = subprocess.Popen(build_command, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-
-                    std_out, std_err = proc.communicate()
-                    std_out = std_out.decode()
-                    std_err = std_err.decode()
-                    print ("\n#### STDOUT ####\n%s\n#### STDERR ####\n%s\n#### End of STDOUT/STDERR ####\n" % (std_out,std_err))
-
-                    if proc.returncode:
-                        failures.append(example_summary)
+                    if cmake:
+                        build_command_seq = ["mbed-tools build -t {} -m {} -c".format(toolchain, target)]
                     else:
+                        build_command_seq = ["mbed-cli compile -t {} -m {} -j {} {}".format(toolchain, target, str(jobs), '-vv' if verbose else '') ]
+                        if profiles:
+                            for profile in profiles:
+                                build_command_seq[0] += " --profile {}".format(profile)
```

Comment:
You can also test all build profiles with `mbed-tools`.
You can use the `--build-type` optional argument of the `build` sub-command.

Reply:
Cheers, good to know that. 
But I as mentioned in Jaeden's comment,  we are not going to introduce build multiple profile at this PR, as this will evolve more CI side code changes. We'll do that as an improvement in near future. In this PR we are only build the default profile which is `development` for now